frb_codegen/src/generator/c/mod.rs

```diff
@@ -1,14 +1,84 @@
-pub fn generate_dummy(func_names: &[String]) -> String {
+use std::path::Path;
+
+use crate::config;
+use crate::utils::{relative_path, BlockIndex, PathExt};
+
+pub fn generate_dummy(
+    config: &config::Opts,
+    configs: &[config::Opts],
```

Comment:
hmm quite confused about this... maybe we can refactor to a better function signature

Reply:
If you think too many "config" is miss leading, then what if change  it  to :
```rust
pub fn generate_dummy(
    configs: &[config::Opts],
    index: usize,
    func_names: &[String],
    c_path_index: usize,
) -> String {
```
But here, it would once again try to fetch current config in side the function. I would suggest just add a doc to clarify what each parameters are used for.

Reply:
(same as below - will revisit later)

Reply:
For this particular one, the configs + index LGTM, since it means "we need to know all configs, and we want to generate a dummy for that index based on other configs"

Reply:
Fixed in 
 https://github.com/fzyzcjy/flutter_rust_bridge/pull/1024/commits/80edccbb1a12c6c21a5de2c78be772536335f670.

Reply:
Weirdly the latest code is still config + configs (not configs + index). Maybe somehow it is accidentally reverted?

Reply:
> Weirdly the latest code is still config + configs (not configs + index). Maybe somehow it is accidentally reverted?
Since there is already a `config` in frb_codegen_multi, there is no need to use the `index` again. But to make the parameters less confusing, the configs is changed to all_configs in https://github.com/fzyzcjy/flutter_rust_bridge/commit/1217983475c2a047aa56593d4bb47da14ee354c1.